src/system_data.jl

```diff
@@ -567,7 +567,7 @@ function _transform_single_time_series!(
             horizon = params.horizon,
             time_series_type = DeterministicSingleTimeSeries,
             scaling_factor_multiplier = get_scaling_factor_multiplier(metadata),
-            internal = get_internal(metadata),
+            internal = InfrastructureSystemsInternal(),
```

Comment:
The previous behavior was incorrect, but was harmless. It caused the metadata of the transformed object to have the same UUID, but no code ever looked at that.